aggregate/housing_security/three_or_more_maintenance_deficiencies.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+from pandas.core.reshape.pivot import crosstab
+from ingest.load_data import load_HVS
+from aggregate.race_assign import HVS_race_assign
+from statistical.calculate_counts import calculate_counts
+from aggregate.clean_aggregated import sort_columns
+from ingest.clean_replicate_weights import rw_cols_clean
+from internal_review.set_internal_review_file import set_internal_review_files
+
+implemeted_years = [2017]
+implemented_geographies = ["Borough"]
+
+
+def count_units_three_or_more_deficiencies(
+    geography_level, year, crosstab_by_race=False, requery=False
```

Comment:
Do you mention needing to layer in the race ethnicity breakdown during standup?

Reply:
Yes I believe I mentioned it. I definitely meant to